openprescribing/pipeline/management/commands/delete_latest_prescribing_import.py

```diff
@@ -0,0 +1,166 @@
+        delete_matrixstore_download(year, month)
+
+
+def verify_year_month(year, month):
+    log = ImportLog.objects.latest_in_category("prescribing")
+    assert log.year == year
+    assert log.month == int(month)
+
+
+def delete_import_record(year, month):
+    import_records = load_import_records()
+    logs = import_records["prescribing"]
+    new_logs = [
+        r for r in logs if f"prescribing_v2/{year}_{month}" not in r["imported_file"]
+    ]
+    assert len(logs) == len(new_logs) + 1
+    import_records["prescribing"] = new_logs
+    dump_import_records(import_records)
+
+
+def mark_task_logs_as_failed(year, month):
+    with open(settings.PIPELINE_METADATA_DIR + "/tasks.json") as f:
+        tasks = json.load(f)
+
+    graph = nx.DiGraph()
+    for task_name, task_def in tasks.items():
+        for dependency_name in task_def.get("dependencies", []):
+            graph.add_edge(dependency_name, task_name)
+
+    convert_task_log = TaskLog.objects.get(
+        task_name="convert_hscic_prescribing",
+        year=year,
+        month=month,
+        status=TaskLog.SUCCESSFUL,
+    )
+
+    for task_name in nx.descendants("convert_hscic_prescribing"):
+        task_log = TaskLog.objects.get(
+            task_name=task_name, year=year, month=month, status=TaskLog.SUCCESSFUL
+        )
+        assert task_log.started_at > convert_task_log.started_at
+        task_log.status = TaskLog.FAILED
+        task_log.save()
+
+    convert_task_log.status = TaskLog.FAILED
+    convert_task_log.save()
+
+
+def delete_fetch_and_import_task_log(year, month):
+    TaskLog.objects.get(task_name="fetch_and_import", year=year, month=month).delete()
+
+
+def delete_import_logs(year, month):
+    ImportLog.objects.filter(category="prescribing", year=year, month=month).delete()
+    ImportLog.objects.filter(category="dashboard_data", year=year, month=month).delete()
+
+
+def delete_prescribing_file_on_filesystem(year, month):
+    path = os.path.join(
+        settings.PIPELINE_DATA_BASEDIR,
+        "prescribing_v2",
+        f"{year}_{month}",
+        f"epd_{year}{month}.csv",
+    )
+    os.remove(path)
+
+
+def delete_prescribing_file_in_storage(year, month):
+    _delete_file_from_storage("hscic/prescribing_v2/2021_10")
+
+
+def delete_temporary_prescribing_bq_table(year, month):
+    try:
+        _delete_table_from_bq("tmp_eu", f"raw_prescribing_data_{year}_{month}")
+    except NotFound:
+        # This is ok, as the table might already have been deleted
+        pass
+
+
+def remove_records_from_bq_table(year, month):
+    client = BQClient("hscic")
+    sql = (
+        f"DELETE FROM ebmdatalab.hscic.prescribing_v2 WHERE month = '{year}-{month}-01'"
+    )
+    client.run_job("query", [sql])
+
+
+def delete_backup_from_storage(year, month):
+    _delete_file_from_storage("backup/prescribing_v2/2021_10")
+
+
+def delete_matrixstore_bq_table(year, month):
+    _delete_file_from_storage("prescribing_export", f"prescribing_{year}_{month}")
```

Comment:
I think this should be:
```suggestion
    _delete_table_from_bq("prescribing_export", f"prescribing_{year}_{month}")
```